sonarqube/resource_sonarqube_group.go

```diff
@@ -59,7 +59,7 @@ func resourceSonarqubeGroup() *schema.Resource {
 
 func resourceSonarqubeGroupCreate(d *schema.ResourceData, m interface{}) error {
 	sonarQubeURL := m.(*ProviderConfiguration).sonarQubeURL
-	sonarQubeURL.Path = "api/user_groups/create"
+	sonarQubeURL.Path += "api/user_groups/create"
```

Comment:
Can we do this instead:

sonarQubeURL.Path = m.(*ProviderConfiguration).sonarQubeURL.Path + "api/user_groups/search"

Reply:
yes, sort of, check above. thanks